hotham/src/components/panel.rs

```diff
@@ -79,63 +60,40 @@ impl PanelButton {
 #[cfg_attr(feature = "cargo-clippy", allow(clippy::too_many_arguments))]
 pub fn add_panel_to_world(
     text: &str,
-    width: u32,
-    height: u32,
-    buttons: Vec<PanelButton>,
+    resolution: vk::Extent2D,
```

Comment:
I think the problem with that is that the resolution is not actually what matters: it's the pixel density of the panel in screenspace, which is a function of the resolution, and distance, and angle, the latter two of which can change. I'm imagining something like a panel attached to your wrist: you might want to render at a higher resolution just in case it's close to the viewer. (I think we need to mipmap the generated texture to reduce aliasing if the resolution is too high, though. I'll try to take a stab at that over the weekend)

I have another patch in a branch that makes `SCALE_FACTOR` a per-panel setting, because it seems odd to me that the resolution of the panel  is what controls the size of UI elements, but I got bogged down into wondering what the units for that value are, and if we should be considering the focal distance of the device and basing it on something related to that.

Reply:
Agreed - this does make a bit more sense, and had thought about the wrist example as well. Let's go with this for now and tackle a better solution in a separate issue.